index.d.ts

```diff
@@ -37,10 +37,6 @@ export interface Action {
  * side-effects. This is what enables exciting features like hot reloading and
  * time travel.
  *
- * Reducers are the most important concept in Redux.
```

Comment:
Is this fair to say?  I feel it's misleading to say one concept or another are the "most important" in Redux.  Perhaps saying they are the primary contribution of, or new concept introduced by, Redux would be more apt.